src/display/font_loader.js

```diff
@@ -113,9 +113,9 @@ class BaseFontLoader {
     unreachable("Abstract method `_queueLoadingCallback`.");
   }
 
-  // eslint-disable-next-line getter-return
   get isFontLoadingAPISupported() {
-    unreachable("Abstract method `isFontLoadingAPISupported`.");
+    const supported = document !== "undefined" && !!document.fonts;
```

Comment:
I think this should be `typeof document`, just like in the old code, because that gives a string return value whereas `document` is most likely the `undefined` type.

Reply:
Sigh, this is a *really* stupid copy-paste error on my part; thanks for spotting it :-)